frontend/src/modules/DashboardModule/components/SummaryCard.jsx

```diff
@@ -54,7 +54,7 @@ export default function AnalyticSummaryCard({
                     justifyContent: 'center',
                   }}
                 >
-                  {tagContent}
+                  {tagContent?tagContent:"NA"}
```

Comment:
Hi @rkmdCodes! please change this to `$ 0.00` as the default value. instead of `NA`. I'm afraid it will confuse the user